src/pages/docs/api-reference/[slug].tsx

```diff
@@ -69,9 +63,15 @@ export const getStaticPaths: GetStaticPaths = () => {
 }
 
 export const getStaticProps: GetStaticProps = ({ params }) => {
+  const matchPath =
+    params?.slug === 'catalog'
+      ? 'get /api/catalog_system/pvt/products/GetProductAndSkuIds'
```

Comment:
Let me know if I'm wrong, but from what I can see these specific docs were chosen as a mock API reference for testing rapidoc-mini. Wouldn't it be interesting to warn about this provisional snippet in the task documentation? Maybe even create a new card in the task board to better guide the future integration of all API references :)

Reply:
Also noticed there are some elements hidden (can't tell if it is behind the header or overflowing something directly from Rapidoc), maybe some top margin would help
![image](https://user-images.githubusercontent.com/39717968/182867822-8e7fcd71-81e0-422b-a6ab-18f150d188bd.png)

Reply:
For the first comment, good call! I will update the task documentation.
As for the second comment, this element is unnecessary for us. Soon this should be deleted from Rapidoc's repository.